notebooks/01-introduction/01-clickstream/01-introduction-clickstream.md

```diff
@@ -0,0 +1,103 @@
+* Which types of visitor achieve high value goals?
+* What actions did someone take prior to a conversion?
+* What conversions take longest to achieve?
+
+These are important questions for a number of organizations, including auction websites, news publishers, and video streaming services.
+
+For example, for news publishers, the front page remains the most important real-estate on a website. Clickstream analytics helps advertising campaign managers understand whether the right demographic is reaching this front page as a campaign runs - and whether the campaign has resulted in longer sessions on the site overall.
+
+Server logs, as are used often by IT, are not enough data to answer today's clickstream questions. Event data from a number of different channels need to be combined together to get the big picture, including telephone interactions, mobile app stats, social media interactions, and even physical store transactions.
+
+Clickstream now concerns the entire customer lifecycle: how the attention of the _visitor_ was grabbed, what _user actions_ took place to achieve _conversion_ through a _session_ - no matter the channel - and how that _visitor_ was made loyal.
+
+> "Clickstream analytics puts us closer to our users, and if you know what your users want, you are better able to serve them."
+
+## Challenges
+
+Analytics on clickstream is generally difficult for these reasons:
+
+* It's hard to collect the data.
+   * Data can come in from multiple sources, so a database needs to be able to draw from multiple real-time and batch streams - both at ingestion and query time.
+   * Data can be very large, so a database must not only be able to scale with the ingestion velocity, but index and compress the data efficiently both for long-term storage and for speedy computation.
+* It's hard to query the data.
+   * The websites, mobile apps, and other channels change over time. A database needs to be adaptable to changing integrations and schemas.
+   * Filtering and statistical needs of uses are often unpredictable, or lead to a large number of reports that have to be maintained. A database needs to be able to cope with a range of query profiles that can be executed in a number of different ways depending on needs.
+   * A broad number of clickstream queries concern distinct counts, especially of visitors. Databases need to have ways to calculate distinct counts quickly.
+   * Statistics very often concern intersections, unions, and differences - visitors who used a channel today but not in the last 30 days. A database needs to be able to carry out set operations quickly enough for the answers to still be relevant.
+
+The index for this guide signposts you to functionality in Apache Druid that many adoptees are using to solve for these challenges.
+
+## Entities and events
+
+Clickstream data tells the story of what visitors came to use a channel, what they were like, what they did, and whether a goal was achieved.
```

Comment:
What do you mean by "what they were like?" Sounds a bit personal? Do you mean that the data captures information about each visitor and if so, what?